### ISSUE TITLE
`CARGO_REGISTRY_TOKEN` is not set

### ISSUE THREAD

**Issue by ocni-dtu** [buehler/semantic-release-cargo#282]
I get the following error when using version 2.3.0:

`An error occurred while running semantic-release: Error: CARGO_REGISTRY_TOKEN is not set.`

Even though I have set the `CARGO_REGISTRY_TOKEN` env var.
Using 2.2.14 it works fine.

**Comment by buehler** [comment]
Hmm... I need to test this.

**Comment by buehler** [comment]
Which version of semantic release are you using?

**Comment by alesbrelih** [comment]
Same here. I've checked and last time it ran successfully it was with the same version of semantic release. So I think problem was introduced with: https://github.com/buehler/semantic-release-cargo/commit/7738929a6cc9f8b02dbcfec9b87de95bdcbcad3b

**Comment by alesbrelih** [comment]
Using this setup:

```yaml
      - name: Semantic Release
        uses: cycjimmy/semantic-release-action@v4
        with:
          extra_plugins: |
            semantic-release-cargo@2.2.14
            @semantic-release/git

```

it works.

**Comment by buehler** [comment]
This should be fixed with 1fb4f8d64157d02800a157f3302026fb7bd24323
